micrometer-core/src/test/java/io/micrometer/core/instrument/binder/cache/GuavaCacheMetricsTest.java

```diff
@@ -78,6 +78,9 @@ void reportExpectedMetrics() {
 
         FunctionCounter failedLoad = fetch(registry, "cache.load", Tags.of("result", "failure")).functionCounter();
         assertThat(failedLoad.count()).isEqualTo(stats.loadExceptionCount());
+
+        Gauge hitRate = fetch(registry, "cache.hit_rate").gauge();
```

Comment:
I wonder if would make sense to put it under cache.gets.rate, since we have already cache.gets.hit /  cache.gets.miss?